tasks/kmt.py

```diff
@@ -127,37 +127,35 @@ def gen_config_from_ci_pipeline(
     """
     Generate a vmconfig.json file with the VMs that failed jobs in the given pipeline.
     """
-    gitlab = Gitlab("DataDog/datadog-agent", str(get_gitlab_token()))
+    repo = get_gitlab_repo()
     vms = set()
     local_arch = full_arch("local")
 
     if pipeline is None:
         raise Exit("Pipeline ID must be provided")
 
     info(f"[+] retrieving all CI jobs for pipeline {pipeline}")
-    for job in gitlab.all_jobs(pipeline):
-        name = job.get("name", "")
+    pipeline_obj = repo.pipelines.get(pipeline)
+    jobs = pipeline_obj.jobs.list(per_page=100, all=True)
+    for job in jobs:
+        name = job.name
 
         if (
             (vcpu is None or memory is None)
             and name.startswith("kernel_matrix_testing_setup_env")
```

Comment:
I can confirm the [names have changed](https://gitlab.ddbuild.io/DataDog/datadog-agent/-/jobs/472524203)
```suggestion
            and name.startswith("kmt_setup_env")
```

Reply:
It has already been made in this PR : https://github.com/DataDog/datadog-agent/pull/24179